app/assets/stylesheets/theme.scss

```diff
@@ -1,6 +1,9 @@
 // Bootstrap theming
 // Ref: https://getbootstrap.com/docs/4.0/getting-started/theming/
 
+// Enable negative margins (disabled by default in Bootstrap)
+$enable-negative-margins: true;
```

Comment:
Can we accomplish the same layout without negative margins, they are generally code smells

Reply:
need to remove whitespace from logo and hero image for this

Reply:
@senbo1 We can edit the image instead